IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -402,6 +402,15 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
     decisionHandler(WKNavigationActionPolicyAllow);
 }
 
+- (void)webView:(WKWebView *)webView didReceiveServerRedirectForProvisionalNavigation:(WKNavigation *)navigation
```

Comment:
Can you shed some light why we're doing this here and not in decidePolicyForNavigationAction?

Reply:
Yes, I wondered about it myself but decided to go with didReceiveServerRedirectForProvisionalNavigation because :
 - It is meant to tell web view that a server redirect was received : [didReceiveServerRedirectForProvisionalNavigation](https://developer.apple.com/documentation/webkit/wknavigationdelegate/1455627-webview?language=objc) 
 - If we decide to handle redirects in decidePolicyForNavigationAction, the only way to determine if a redirect was received is to check if [WKNavigationType](https://developer.apple.com/documentation/webkit/wknavigationtype?language=objc) is [WKNavigationTypeOther](https://developer.apple.com/documentation/webkit/wknavigationtype/wknavigationtypeother?language=objc), which may not necessarily mean a redirect.